app/src/main/java/atlant/moviesapp/activity/MovieDetailsActivity.java

```diff
@@ -195,7 +284,7 @@ public void onLongClick(View view, int position) {
     public void showCrew(List<Crew> crew) {
         String directorsString = "";
```

Comment:
Later, when you have time, all String manipulation, concatenation etc.. should be in a StringUtlis, StringHelper utility class. So please create a StringUtils utility Java class to handle all String manipulations.